app/Domain/Integrations/Repositories/EloquentIntegrationRepository.php

```diff
@@ -50,9 +52,24 @@ public function update(Integration $integration): void
             'status' => $integration->status,
             'partner_status' => $integration->partnerStatus,
             'key_visibility' => $integration->getKeyVisibility(),
+            'reminder_email_sent' => $integration->reminderEmailSent,
         ]);
     }
 
+    /** @return Collection<Integration> */
+    public function getIntegrationsThatHaveNotBeenActivatedYetByType(IntegrationType $type, int $months): Collection
```

Comment:
I think something like `getDraftsByTypeAndMonths` covers the implementation better.

Reply:
Renamed it to "getDraftsByTypeAndOlderThenMonthsAgo" to communicate that it looks at integrations X months ago.